Makefile

```diff
@@ -2,6 +2,18 @@ DEFAULT_GOAL: help
 SHELL := /bin/bash
 PWD := $(shell pwd)
 
+.PHONY: images
+images:
+	$(MAKE) -C securedrop images
+
+.PHONY: test
```

Comment:
lets prepend this with `app-` . we have a number of different `test` suites and we need to make it more obvious which one we are referring to. also needs documentation for commands.

Reply:
or you can cherry pick `8e43edb1` in :)

Reply:
done. Thanks!